### ISSUE TITLE
Implement first net architecture using LSTM

### ISSUE THREAD

**Issue by picost** [picost/udl_tvscript_project#5]
Implement a first architecture of neural network for the project using an LSTM combined with an embedding layer.

**Comment by picost** [comment]
Rq: the criterion used for training in an imposed cell is CrossEntropyLoss. The network should therefore return raw scores as this loss already includes a log-softmax function.
